auth-web/src/components/auth/manage-business/EntityManagement.vue

```diff
@@ -294,6 +295,7 @@
         :isStaffOrSbcStaff="isStaffAccount || isSbcStaffAccount"
         :userFirstName="currentUser.firstName"
         :userLastName="currentUser.lastName"
+        :initial-business-identifier="initialBusinessIdentifierManageBusinessDialog"
```

Comment:
`this.businessIdentifier`?

Reply:
I dont know, I could name it like that, but am afraid someone might think it is default for the page or smth.

Reply:
`:initial-business-identifier="businessIdentifier"`